tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrerTest.xtend

```diff
@@ -0,0 +1,162 @@
+package org.testeditor.tcl.dsl.jvmmodel
+
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.dsl.common.testing.DummyFixture
+
+class TclJvmModelInferrerTest extends AbstractTclGeneratorIntegrationTest {
+
+	@Before
+	def void parseAmlModel() {
+		parseAml(DummyFixture.amlModel + '''
+			component type dummyComponentType {
```

Comment:
component type Application of the amlModel should be enough. Additionally defining dummyComponent and dummyComponentType is not necessary, is it?